src/test/java/org/zalando/riptide/LambdaTest.java

```diff
@@ -0,0 +1,107 @@
+package org.zalando.riptide;
+
+/*
+ * ⁣​
+ * Riptide
+ * ⁣⁣
+ * Copyright (C) 2015 - 2016 Zalando SE
+ * ⁣⁣
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ​⁣
+ */
+
+import com.google.common.reflect.TypeToken;
+import org.junit.Test;
+import org.springframework.http.ResponseEntity;
+
+import static org.zalando.riptide.Conditions.anyStatus;
+import static org.zalando.riptide.Selectors.status;
+
+/**
+ * Tests whether all callbacks/functions can be supplied as lambdas. If this class compiles everything is already fine.
```

Comment:
Funny. Compile time testing ... didn't even need to annotate with \@Test.

But, do we really need to test this explicitly?